src/main/webapp/data-visualization.js

```diff
@@ -21,26 +21,34 @@ async function getGeoData(location, isCountyQuery) {
   }
 }
 
-// Display amCharts and geoJson visulizations and create data table
-// for given data.
+// Display amCharts and geoJson visulizations for given data.
 async function displayVisualization(censusDataArray, description,
   location, isCountyQuery) {
+  document.getElementById('colors').style.display = 'block';
+  getColor();
```

Comment:
getColor (now named setColor) also adds the default blue color to the cache, so I think I will leave as is.

Reply:
you can avoid yet another call to localStorage.getItem if you just return the value from setColor().

so instead of setColor();
const color = localStorage.getItem('color');

you can simply have const color = setColor();

Reply:
your setColor function can be like this.

function setColor() {
  const color = localStorage.getItem('color') || '#0071bd';
  document.getElementById('setColor').value = color;
  return color;
}

Reply:
I see, this definitely alleviates the amount of calls to localStorage. Fixed.